doc/telescope.txt

```diff
@@ -1868,6 +1868,34 @@ resolver.resolve_anchor_pos()                  *resolver.resolve_anchor_pos()*
 
 Actions functions that are useful for people creating their own mappings.
 
+Actions can be either normal functions that expect the prompt_bufnr as first
```

Comment:
I'd suggest to extend this paragraph with something like

> The `prompt_bufnr` denotes the unique identifier of your picker to access the picker state. Beyond the picker state, telescope.nvim also retains a global state. In practice, users most commonly access from both picker and global state via 
local action_state = require "telescope.actions.state"
local current_picker = action_state.get_current_picker(prompt_bufnr) -- picker state
local entry = action_state.get_selected_entry() -- global state

I guess part on global state is debatable , but I'd suggest to add how to most commonly access relevant info of telescope(-picker) state here (= the code segment). Maybe it's worthwhile adding a small example function here as well, even if it's just printing the `entry.value`. I'm 100% certain that we surely mention that already somewhere, but telescope is too big/scattered and users don't get across all the points of information :sweat_smile:

Reply:
that seems like a good idea. I'll try to integrate it. Thanks :)